.github/workflows/ci.yml

```diff
@@ -40,6 +51,6 @@ jobs:
       with:
         cache: 'npm'
     - run: npm ci
-    - run: npm test
```

Comment:
So you no longer need to run `npm test` 👀

Reply:
it's calling `bin/ci-run`, but actually I should remove the script from `package.json` since I've removed that file